flybywire-aircraft-a320-neo/es-ES.locPak

```diff
@@ -12,7 +12,7 @@
       "A32NX.LOADING.TIPS.A320NEO_001": "El FlyByWire A32NX es una colaboración entre cientos de colaboradores individuales.",
       "A32NX.LOADING.TIPS.A320NEO_002": "¿Sabías que? El A320 puede aterrizar con vientos cruzados de hasta 38 nudos y con vientos de cola de hasta 15 nudos.",
       "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire recibe su nombre de los controles fly-by-wire introducidos por primera vez en el Airbus A320.",
-      "A32NX.LOADING.TIPS.A320NEO_004": "Hay 9693 aviones de la familia A320 construidos. ¡Eso es casi tanto como la población de Tuvalu!",
+      "A32NX.LOADING.TIPS.A320NEO_004": "Hay 9873 aviones de la familia A320 construidos. ¡Eso es casi tanto como la población de Tuvalu!",
```

Comment:
```suggestion
      "A32NX.LOADING.TIPS.A320NEO_004": "Hay cerca de 9873 aviones de la familia A320 construidos. ¡Eso es casi tanto como la población de Tuvalu!",
```

Reply:
"¡Se han construido más de 9900 aviones de la familia A320!" is what i became